<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port_line"/>
       </item>
       <item>
        <widget class="QLabel" name="testPort">
         <property name="text">
          <string>Address IP: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="address_line"/>
       </item>
       <item>
        <widget class="QPushButton" name="ConnectToServer">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="callBtn">
       <property name="text">
        <string>call</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hangUp">
       <property name="text">
        <string>Hangup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
